src/utils/processArguments.js

```diff
@@ -0,0 +1,27 @@
+const argv = require('yargs')
+    .option('o', {
+        alias: 'output',
+        describe: 'Output file path with miragejs generated code.',
+        type: 'string',
+        nargs: 1,
+        demand: true,
+        demand: 'An output path is required',
```

Comment:
Are we sure the `demand: true` is doing something? Saying this because having an object with repeated keys keeps the last one, and it is a confusing practice.

We'll have to install a linter and add a CI to this so we're warned about this and we don't need to be concerned about it. 

In my opinion, the linter is more important as we can, for now, run it locally

Reply:
You're right! 🤦‍♂ We really need a linter 😄